packages/InteractiveProfilingTool-Tests.package/InteractiveProfilingToolTestsServiceProvider.class/class/initialize.st

```diff
@@ -0,0 +1,3 @@
+initialization
```

Comment:
What does the `InteractiveProfilingToolTestsServiceProvider` class do and what does this method do?

Reply:
That is a good question. We asked us the same...
This is in the repo by fault. I will remove it soon.